marine.c

```diff
@@ -184,6 +185,9 @@ static frame_data prev_cap_frame;
 static guint32 epan_auto_reset_count = 20000; // TODO make this configurable
 static gboolean epan_auto_reset = TRUE;
 
+const unsigned int ETHERNET_ENCAP = 1;
```

Comment:
Can't you just declare and assign in `marine.h` and remove these lines from `marine.c`?
Seems nicer to me

Reply:
You cannot, C doesn't let you put an initializer for an extern variable (and you do need to define it as extern in `marine.h`)